### ISSUE TITLE
Crash when trying to eat 3/4 piece of cheese

### ISSUE THREAD

**Issue by chekalin87** [anegostudios/VintageStory-Issues#719]
**Game Version:** 1.14.2
**Platform:** Windows
**Modded:**  Yes

### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.14.2 (Stable)12/28/2020 6:48:03 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.GameContent.ItemCheese.OnHeldInteractStart(ItemSlot slot, EntityAgent byEntity, BlockSelection blockSel, EntitySelection entitySel, Boolean firstEvent, EnumHandHandling& handling)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel, EnumHandInteract useType, EnumHandHandling& handling)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsNoBlockSelected(Single dt)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   at _uWwiuXC8cGTH1JnsWiUY9FUaXOo._X2niSFpb8ndF6zWsCcobp5QBcvO(Single )
   at _ydpin5JFjepNgd3JOZRb2z5I5Fg._LvRZFcsDWyPwKLB1F2UK7fQ2N6d(Single )
   at _ydpin5JFjepNgd3JOZRb2z5I5Fg._GO2qaUrk9dl0IceTLiA7P4LcpAB(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _Ine8FeGTtDXciSIfaZMHMwCLKGBA._8puLW8J44fiq3A5hthQTfB8WNxc(_SbVlAI6dJIZ4JRt0da0lu23DF14 , String[] )
   at _lErb0L2e9E9htuT2VVnAsXIZqc7A._8puLW8J44fiq3A5hthQTfB8WNxc(ThreadStart )
-------------------------------

```

**Comment by tyronx** [comment]
hm, i cannot reproduce this. a 3/4 slice is not edible for me

**Comment by chekalin87** [comment]
Sorry! This happens when you try to eat cheese with the shift pressed.
(even with regular 1/4 pieces)

**Comment by tyronx** [comment]
Oh wow, thanks. Can reproduce that now. Fixed for v1.14.3.!
